### PR TITLE
docs: improve package visibility

### PR DESCRIPTION
**Pull request by jackshen310**
## Release Notes

- **New Features**
	- Added a `CONTRIBUTING.md` file with guidelines for contributors, including development setup and pull request processes.
	- Introduced a new `test_stream_with_signal` function to enhance streaming capabilities with abort signal handling.

- **Improvements**
	- Restructured and updated the `README.md` files across packages for better clarity and organization.
	- Added badges to `README.md` files for quick reference on version, downloads, and licensing.
	- Enhanced `package.json` files with detailed descriptions and keywords for improved discoverability.
	- Introduced the MIT License to the project, outlining user permissions and limitations. 

- **Bug Fixes**
	- Updated instructions for local development setup to clarify processes.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Contributing to coze-js
+
+We love your input! We want to make contributing to coze-js as easy and transparent as possible, whether it's:
```
Comment (Tecvan-fe) [inline CONTRIBUTING.md:3]: 好像有点问题，`we want to xxx` 是我们想做什么什么的意思，这里想表达的应该是”希望你做什么什么“？

Comment (jsongo) [reply CONTRIBUTING.md:3]: Try this: https://mozillascience.github.io/working-open-workshop/contributing/

**Review by jsongo — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (jackshen310) [reply CONTRIBUTING.md:3]: 这个我重新一下

Comment (jackshen310) [reply CONTRIBUTING.md]: @jsongo @Tecvan-fe 已经修改，再看看

**Pull request merged**